### DISCUSSION TITLE
Not Able to Access Github Developer Pack Benifits

### DISCUSSION THREAD

**Discussion by guranshdeol** [community/community#38860] [GitHub Education]
I have been enrolled in GitHub Student Developer Pack since last year, but now i am not able to access its benefits even though i am verified.
Yesterday i noticed on my Github Profile that i am not a PRO user anymore.  When i opened Github Education site to check my Developer Pack, Surprisingly all the benefits were not visible.

This page is showing me to verify as a student.
![image](https://user-images.githubusercontent.com/91736425/201339781-779eee28-907c-4adc-accb-9ae3ca2bb1b5.png)

But the next one is saying I am already verified and can access my GitHub Pack.
![image (1)](https://user-images.githubusercontent.com/91736425/201339923-77c67b85-882a-4f1a-9e8f-7718f18c9cc8.png)
When I head towards Benefits, it again shows sign up for Developer pack.

& if I proceed for re-verification, i am also not able to do so aswell.
![image (2)](https://user-images.githubusercontent.com/91736425/201340695-21d6342c-2f6a-4bc6-a00c-9d39e79e9420.png)

Can Anyone help me with this issue.

**Comment by FuckingDonbass** [comment]
I have the same problem

**Comment by samus-aran** [comment]
Hi @guranshdeol 👋 

Are you still having this issue accessing your Global Campus Benefits?

**Reply by guranshdeol** [reply]
its been resolved now thanks for your help😊
